app/src/main/java/com/shagalalab/constitution/list/adapter/ItemViewHolder.kt

```diff
@@ -0,0 +1,24 @@
+package com.shagalalab.constitution.list.adapter
+
+import android.view.View
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+import com.shagalalab.constitution.R
+import com.shagalalab.constitution.list.model.Item
+
+class ItemViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
+
+    var title: TextView = itemView.findViewById(R.id.title_text)
+    var description: TextView = itemView.findViewById(R.id.description_text)
+
+    fun populateModel(model: Item, itemClickListener: ItemClickListener) {
+        title.text = model.title
+        description.text = model.description
+        if (model.description == "") {
+            description.visibility = View.GONE
```

Comment:
There is a `isVisible` extension function for `View` to which you pass boolean statement, then there is no need for `if` condition below.